pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -843,6 +843,8 @@ type S3StorageProvider struct {
 	Prefix string `json:"prefix,omitempty"`
 	// SSE Sever-Side Encryption.
 	SSE string `json:"sse,omitempty"`
+	// Options Rclone options for backup and restore with mydumper and lightning.
+	Options string `json:"options,omitempty"`
```

Comment:
why not use string slice?

Reply:
I'm not sure whether expose args by a string is ok to users. Can we import the rclone [configFlag](https://github.com/rclone/rclone/blob/ab6ed256e529a8c134c0ff0ff0848111919d1795/fs/config/configflags/configflags.go) as rcloneConfig in the `Backup` and `Restore` Spec?

Reply:
That would be too much config fields. Since mydumper+lightning is not our focus already, I think it's OK to support like this.
I will change it to string slice.

Reply:
I think we don't have to set all the configuration of rclone, just use the configuration we need.